credoai/evaluators/evaluator.py

```diff
@@ -104,11 +105,20 @@ def get_container_info(self, labels: dict = None, metadata: dict = None):
         return info
 
     def _base_container_info(self):
-        return {"labels": {"evaluator": self.name}, "metadata": self._get_artifacts()}
+        meta = {**self.metadata, **self._get_artifacts()}
+        labels = {"evaluator": self.name}
+        if "dataset_type" in meta:
+            labels["dataset_type"] = meta["dataset_type"]
+        return {"labels": labels, "metadata": meta}
 
     def _get_artifacts(self):
         artifacts = {}
-        save_keys = {"model": "model_name"}
+        save_keys = {
+            "model": "model_name",
+            "data": "data_name",
+            "assessment_data": "assessment_data_name",
+            "training_data": "training_data_name",
+        }
         for k in self.artifact_keys:
             save_key = save_keys.get(k, k)
             try:
```

Comment:
Just curious, in which case does this throw back an error?
`save_key` always returns something and `self.__dict__[k].name` should
as well since `k` is part of the artifact keys which are created when
we are passing the artifacts to the evaluator.

Reply:
I think it was sensitive features.